src/widgets/icons.js

```diff
@@ -48,3 +49,13 @@ export const CloseIcon = React.memo(() => <IonIcon name="md-close" size={36} col
 export const Expand = React.memo(() => (
   <FAIcon name="external-link" size={16} color={SUSSOL_ORANGE} />
 ));
+
+export const Confirm = ({ style }) => <FAIcon name="check-circle" style={style} />;
```

Comment:
Needs export?

Reply:
Do you have to separate them out from the `memo` call to get defaultProps and propTypes in there?

Reply:
this is really dumb I don't know f0bd923a47539985935a86295b998d5abfaaab23